.clang-format

```diff
@@ -11,7 +11,7 @@ AlignOperands: true
 AlignTrailingComments: false
 AllowAllParametersOfDeclarationOnNextLine: false
 AllowShortBlocksOnASingleLine: Never
-AllowShortFunctionsOnASingleLine: Inline
+AllowShortFunctionsOnASingleLine: All

```

Comment:
If you load the full diff on GitHub your browser might die. I recommend reviewing it offline.

---

Curiously there's no difference between `Inline` and `All` in clang-format 13 (which is what we currently use), but there is a difference in the upcoming 14! This change fixes the upcoming incompatibility ahead of time (and was needed in order to run clang-format 14 on this project).